modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -380,10 +380,12 @@ in {
       source = configFile;
       onChange = ''
         swaySocket=''${XDG_RUNTIME_DIR:-/run/user/$UID}/sway-ipc.$UID.*.sock
-        if [ -S $swaySocket ]; then
-          echo "Reloading sway"
-          $DRY_RUN_CMD ${cfg.package}/bin/swaymsg -s $swaySocket reload
-        fi
+        for socket in $swaySocket; do
+          if [ -S $socket ]; then
+            echo "Reloading sway"
```

Comment:
Hmm, something I just came across is that this `echo`s for every socket tested. Would it be acceptable to echo after the reload occurs? E.g. `reload &&  echo "Reloading sway" && break`.